filter_plugins/netdata.py

```diff
@@ -19,3 +20,15 @@ def aws_creds(self, data, type_='access'):
         if type_ == 'secret':
             return config.get('default', 'aws_secret_access_key')
         return config.get('default', 'aws_access_key_id')
+
+    def redis_roles(self, services):
+        """ Organise redis services into clusters for the redis collector """
+        tmp = dict()
+        res = []
+        for service in services:
+            tmp.setdefault(service['role'], [])
```

Comment:
You can use a [defaultdict](https://docs.python.org/2/library/collections.html#collections.defaultdict)

Reply:
Let's avoid additional imports. Collections structures are meant for performance, we clearly don't need this here.

Reply:
That'd be for code clarity. What's the problem with "an additional import"?

Reply:
won't fix